custom/icds_reports/static/js/directives/download/download.directive.js

```diff
@@ -441,7 +441,10 @@ function DownloadController($rootScope, $location, locationHierarchy, locationsS
     vm.hasErrors = function () {
         var beneficiaryListErrors = vm.isChildBeneficiaryListSelected() && (vm.selectedFilterOptions().length === 0 || !vm.isDistrictOrBelowSelected());
         var incentiveReportErrors = vm.isIncentiveReportSelected() && !vm.isStateSelected();
-        var ladySupervisorReportErrors = vm.isLadySupervisorSelected() && !vm.isStateSelected();
+        var ladySupervisorReportErrors = false;
+        if (!vm.haveAccessToFeatures) {
```

Comment:
Is this feature supposed to under FF?Yes then. Should not it be `if (vm.haveAccessToFeatures)` instead of `if (!vm.haveAccessToFeatures)`?

Reply:
if user is not under ff, then he will not be able to download as per current functinality. This is has errors method which checks for restriction

Reply:
I see ok